src/Sarif.FunctionalTests/SpecExamples/RecommendedWithoutSource.sarif

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "1.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "BinaryScanner",
+        "semanticVersion": "1.0.1"
+      },
+      "files": {
+        "file:///user/builder/work/bin/example": {
+          "mimeType": "application/vnd.microsoft.portable-executable"
+        }
+      },
+      "logicalLocations": {
+        "Example.Worker.DoWork": {
```

Comment:
Suggest that you reverse the ordering of the logical locations building them out from least to most deeply nested. Communicates the parenting more clearly. 

+1 nice example.